pkg/interface/src/logic/lib/omnibox.js

```diff
@@ -4,7 +4,7 @@ import { cite } from '~/logic/lib/util';
   const indexes = new Map([
     ['commands', []],
     ['subscriptions', []],
-    ['groups', []],
+    ['landscape', []],
```

Comment:
this has ramifications for the Omnibox component, since we return this object to the state there. It will also rename the header to "Landscape" in the results, so do we really need to rename this?

Reply:
Uh no we don't need to rename it. Idk how omnibox works...

Reply:
tl;dr we basically make a list here corresponding to headers in the results, as well as to categories in the returned map

https://github.com/urbit/urbit/blob/cbe58f05199154bfc34caf8f9789485d9669f728/pkg/interface/src/views/components/leap/Omnibox.js#L54-L56

and then iterate through it

https://github.com/urbit/urbit/blob/cbe58f05199154bfc34caf8f9789485d9669f728/pkg/interface/src/views/components/leap/Omnibox.js#L149-L161

and make header'd results with it

https://github.com/urbit/urbit/blob/cbe58f05199154bfc34caf8f9789485d9669f728/pkg/interface/src/views/components/leap/Omnibox.js#L225-L251

Reply:
Arguably you could replace `getSearchedCategories` with the keys of `state.index` and it would be fine tho — but it would still write "landscape" as a header heh

Reply:
Uh... can you do this omnibox part? Seems delicate

Reply:
incoming